CO2P/donations.yml

```diff
@@ -0,0 +1,55 @@
+---
+
+
+- ticker: CO2P
+
+
+  recipient: 
+    name: GREEN FOREST FUND e.V.
+    address: Rehovotstraße 5, 69115 Heidelberg, Germany
+    tel: 
```

Comment:
If you could add the phone number here the rest looks good.

I found their number on their [LinkedIn](https://www.linkedin.com/company/greenforestfund/about/?viewAsMember=true) and [ZoomInfo](https://www.zoominfo.com/c/the-green-forest-fund-ev/466547349)

You may also need to update your repo with the main branch to make sure it is not out of date. Instructions are in the [README.md](https://github.com/CardanoMDP/Donations#keeping-your-repository-up-to-date-with-the-upstream-repository-cardanomdpdonations)